assets/js/components/Dashboard/AlertDetails.tsx

```diff
@@ -50,6 +57,10 @@ const AlertDetails: ComponentType = () => {
     initialPlacesListState
   );
 
+  const validAlertId = contextState.allAPIAlertIds.length
```

Comment:
You could change this to a one-liner (well kinda, formatting may make it two lines).

`const validAlertId = contextState.allAPIAlertIds.find((alert) => alert !== undefined && alert === id);`

Reply:
Ope! I didn't realize `find` would work on an empty array and just return `undefined`. In that case, can condense even further: 
`const validAlertId = contextState.allAPIAlertIds.find(alert => alert === id);`